examples/usage/cooldown.rs

```diff
@@ -0,0 +1,163 @@
+//! Demonstrates implementing a cooldown in UI.
+//! We create three buttons with 2, 1, and 5 seconds cooldown.
```

Comment:
As this is a usage example, I think it would be great to motivate it a bit more, i.e. give some examples on what you might want to use it for